src/graphics/program-lib/chunks/common/frag/envAtlas.js

```diff
@@ -0,0 +1,23 @@
+export default /* glsl */`
+// the envAtlas is fixed at 512 pixels. every equirect is generated with 1 pixel boundary.
+const float atlasSize = 512.0;
```

Comment:
not related to this PR, but don't we handle larger atlases now?

Reply:
We do, but actually still layout them out as if they were 512.